libdevcore/JSON.h

```diff
@@ -23,22 +23,97 @@
 #pragma once
 
 #include <json/json.h>
+#include <sstream>
+#include <memory>
 
 namespace dev
 {
 
+namespace json
+{
+
+/// StreamWriterBuilder that can be constructed with specific settings
+class StreamWriterBuilder : public Json::StreamWriterBuilder {
+public:
+	explicit StreamWriterBuilder(const std::map<std::string, std::string> &_settings)
+	{
+		for (auto& iter : _settings)
+		{
+			this->settings_[iter.first] = iter.second;
+		}
+	}
+};
+
+/// CharReaderBuilder with strict-mode settings
+class StrictModeCharReaderBuilder : public Json::CharReaderBuilder {
+public:
+	StrictModeCharReaderBuilder()
+	{
+		Json::CharReaderBuilder::strictMode(&this->settings_);
+	}
+};
+
+/// Serialise the JSON object (@a _input) with specific builder (@a _builder)
+/// \param _input JSON input string
+/// \param _builder StreamWriterBuilder that is used to create new Json::StreamWriter
+/// \return serialized json object
+inline std::string print(Json::Value const &_input, Json::StreamWriterBuilder &_builder) {
```

Comment:
Please put `{` on its own line and move `&` to the left: `Json::Value const& _input`

Reply:
Ok.